hoot-services/src/main/java/hoot/services/utils/DbUtils.java

```diff
@@ -534,38 +541,125 @@ public static void upsertCommandStatus(CommandResult commandResult){
             else {
                 String queryUpdate = String.format(
                         "UPDATE command_status " +
-                        "SET stdout = '%s', stderr = '%s' " +
+                        "SET stdout = '%s', stderr = '%s', percent_complete = '%.2f' " +
```

Comment:
Yea i made it based off the job status table, but you're right that it should be an int. Core isnt even returning a decimal and it would be way too next level to expect that